### PR TITLE
samples: Add seek subscription sample

### PR DESCRIPTION
**Pull request by tmdiep**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-pubsublite/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by tmdiep** [comment]
Note: this sample is for a pre-release feature.

File: samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  public static void main(String... args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String cloudRegion = "your-cloud-region";
+    char zoneId = 'b';
+    // Choose an existing subscription.
+    String subscriptionId = "your-subscription-id";
+    long projectNumber = Long.parseLong("123456789");
+
+    // Optional parameters.
+    SeekTarget target = SeekTarget.of(BacklogLocation.BEGINNING);
```
Comment (anguillanneuf) [inline samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java:44]: Describe the goal of the seek operation in comments, other BacklogLocation and briefly the usage.

File: samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    AdminClientSettings adminClientSettings =
+        AdminClientSettings.newBuilder().setRegion(CloudRegion.of(cloudRegion)).build();
+
+    try (AdminClient adminClient = AdminClient.create(adminClientSettings)) {
+      OperationFuture<SeekSubscriptionResponse, OperationMetadata> seekFuture =
+          adminClient.seekSubscription(subscriptionPath, target);
+      System.out.println("Seek operation " + seekFuture.getName() + " initiated successfully.");
+
+      // Optional: Wait for the seek operation to complete, which indicates when subscribers for all
+      // partitions are receiving messages from the seek target.
```
Comment (anguillanneuf) [inline samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java:76]: Perhaps say some of this in the TODO(Developer) section when they set SeekTarget.

**Review by anguillanneuf — CHANGES_REQUESTED** [review]
Thanks Thanh-Mai. Just add more comments about SeekTarget.

File: samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java (hunk 1 above)
Comment (tmdiep) [reply samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java]: Done

File: samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java (hunk 2 above)
Comment (tmdiep) [reply samples/snippets/src/main/java/pubsublite/SeekSubscriptionExample.java]: Done

**Comment by tmdiep** [comment]
Thanks Tianzi. It is indeed clearer with more comments.

**Review by anguillanneuf — APPROVED** [review]

**Pull request merged**